types.go

```diff
@@ -0,0 +1,85 @@
+package weave
+
+import (
+	"github.com/iov-one/weave/errors"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"strings"
+)
+
+// CommitInfoFromABCI converts abci commit info to weave native type.
+// This struct represents validator signatures on the current block.
+func CommitInfoFromABCI(info abci.LastCommitInfo) CommitInfo {
```

Comment:
Nice functions. Can you add them to `abci.go`, which is where I try to keep the abci specific code?